packages/validation/src/Call/index.ts

```diff
@@ -24,6 +24,18 @@ const firstStepCreateCallValidationSchema = Yup.object().shape({
         'End call date can not be before start call date.'
       );
     }),
+  endCallInternal: Yup.date()
+    .typeError(TYPE_ERR_INVALID_DATE_TIME)
+    .when('endCall', (endCall: Date, schema: Yup.DateSchema) => {
+      if (!isValidDate(endCall)) {
+        return schema;
+      }
+
+      return schema.min(
+        endCall,
+        'Internal call end date can not be before  call end date.'
```

Comment:
As I can see there are multiple spaces here between "before" and "call". This is just a small fix. Otherwise it looks good.